### PR TITLE
added blog content for openspace

### PR DESCRIPTION
**Pull request by shankig**
Added blog content for open space, please review and merge
@vnbang2003 @kracekumar

### CODE REVIEW

File: content/blog/open-space-registrations.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Title: PyCon India have started accepting proposals for Open Space
+Date: 2015-09-23 10:00
+Category: 2015
+Slug: open-space-registrations
+Summary: PyCon India have started accepting proposals for Open Space
+
+PyCon has always been more than just a conference. It has been a place where people who have similar interests find each other and have great conversations.
```
Comment (harisibrahimkv) [inline content/blog/open-space-registrations.md]: I think you should use `PyCon India`. And do link it to http://in.pycon.org/

File: content/blog/open-space-registrations.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+This time, we want to make room for more such conversations. This time, we are going to make Open Spaces more exciting. Attending PyCon India 2015? Share your experience by giving an Open Space talk.
+
+**What are Open Space?**
+
```
Comment (harisibrahimkv) [inline content/blog/open-space-registrations.md]: `What is an Open space?` Or `What are Open Spaces?`

File: content/blog/open-space-registrations.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+Title: PyCon India have started accepting proposals for Open Space
+Date: 2015-09-23 10:00
```
Comment (harisibrahimkv) [inline content/blog/open-space-registrations.md]: Shouldn't it be PyCon India `has`?

**Comment by shankig** [comment]
Updated

**Pull request merged**